src/lib_ccx/ccx_common_option.c

```diff
@@ -140,7 +140,7 @@ void init_options (struct ccx_s_options *options)
 	options->settings_dtvcc.print_file_reports = 1;
 	options->settings_dtvcc.no_rollup = 0;
 	options->settings_dtvcc.report = NULL;
-	memset(options->settings_dtvcc.services_enabled, 0, CCX_DTVCC_MAX_SERVICES);
+	memset(options->settings_dtvcc.services_enabled, 0, CCX_DTVCC_MAX_SERVICES * sizeof(int));
```

Comment:
where does this (int) come from? I mean, I know what you mean, but you're adding there some magic I'd say. What if options->settings_dtvcc.services_enabled was something else? Your memset would break.

Reply:
settings_dtvcc.services_enabled is an array of type int. To fully change its bytes to 0 we need not only size of the array but also size of data stored.
May change to sizeof(settings_dtvcc.services_enabled), though it's too big (as I see)

Reply:
That's my point - if you are using sizeof you can as well use sizeof (item)*num_items rather than sizeof (type of item you know because you've inspected the code)*num_items.

Reply:
Got it:)